Subject: [PATCH] qatlib, qatzip: move autogen.sh shebangs to the first line

Both bundled autogen.sh scripts have their shebang below the license
header, causing an exec format error when CMake executes them directly.
Move the shebangs to the first line, which also allows patchShebangs to
rewrite the interpreter paths.

Upstream fixes:
https://github.com/intel/qatlib/pull/130
https://github.com/intel/QATzip/issues/143

diff --git a/src/qatlib/autogen.sh b/src/qatlib/autogen.sh
--- a/src/qatlib/autogen.sh
+++ b/src/qatlib/autogen.sh
@@ -1,3 +1,4 @@
+#!/bin/sh
 #################################################################
 #
 #   BSD LICENSE
@@ -34,5 +35,4 @@
 #
 #################################################################
-#!/bin/sh
 
 autoreconf -if
diff --git a/src/qatzip/autogen.sh b/src/qatzip/autogen.sh
--- a/src/qatzip/autogen.sh
+++ b/src/qatzip/autogen.sh
@@ -1,3 +1,4 @@
+#!/bin/sh
 ################################################################
 #   BSD LICENSE
 #
@@ -32,5 +33,4 @@
 #
 ################################################################
-#!/bin/sh
 mkdir -p m4
 autoreconf -vif
